config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -8,10 +8,21 @@ en:
             my_student_header: 'My Student Submissions'
             student_header: 'Student Submissions'
             other_header: 'Other Submissions'
+            publish: 'Publish Grades'
+            confirm_publish: |
+
+              THIS ACTION IS IRREVERSIBLE
+
+              All students will see their grades.
```

Comment:
Eh, that looks not very dangerous.  How about `All graded submissions will be published and students will see the grades`